habitatstreams/src/22_progress_evaluation_3260.Rmd

```diff
@@ -0,0 +1,522 @@
+         position_change = FALSE,
+         distance = 0)
+  
+sample_assessments_3260_update <- sample_assessments_3260_original %>%
+  bind_rows(sample_assessments_3260_replaced) %>%
+  mutate(segment_id = sampling_unit_code,
+         sampling_unit_code = str_c(grts_ranking, "_3260"),
+         x = round(x, 2),
+         y = round(y, 2),
+         sac_streams = factor(sac_streams))
+       
+```
+
+
+### mhq_streams_refpoints
+
+De tabel 'mhq_streams_refpoints' geeft een overzicht van de zogenaamde referentiepunten voor alle steekproefeenheden in het meetnet voor habitattype 3260. Elke steekproefeenheid heeft een referentiepunt waar naar toe genavigeerd wordt om de meting uit te voeren. Bij stromende wateren is het referentiepunt het startpunt van het 100 meter segment (en meer specifiek het meest stroomafwaarts gelegen punt van het segment).
+
+De tabel bestaat uit volgende variabelen:
+
++ scheme: code voor het meetnet;
++ sampling_unit_code: code voor de steekproefeenheid;
++ segment_id: id van het riviersegment;
++ point_code: code voor het referentiepunt;
++ type_target: doelhabitat;
++ sac_streams: positie t.o.v. speciale beschermingszone (SBZ) met onderscheid tussen:
+  + buiten SBZ ('outside')
+  + intersectie met SBZ of stroomafwaarts van SBZ ('intersecting or downstream');
++ grts_ranking: grts-rangnummer van grts-hok waarbinnen referentiepunt gelegen is;
++ grts_ranking_draw: grts-rangnummer op basis waarvan de steekproeftrekking is gebeurd. In de meeste gevallen is dit gelijk aan de grts_ranking, maar als een referentiepunt verplaatst wordt omdat het originele referentiepunt niet geschikt is, behouden we de grts-ranking van het originele referentiepunt voor grts_ranking_draw ook als het nieuwe punt in een ander grts-hok gelegen is (met een andere grts-ranking);
++ legacy_site: gaat het om een al langer bestaand meetpunt? 
++ x- en y-coördinaten van het referentiepunt;
++ db_ref: een andere referentie voor een steekproefeenheid die in een databank gebruikt wordt.
+
+```{r}
+mhq_streams_refpoints <- sample_assessments_3260_update %>%
+  mutate(legacy_site = FALSE,
+         db_ref = field_code) %>%
+  select(scheme, sampling_unit_code, segment_id, point_code, type_target, sac_streams, grts_ranking, grts_ranking_draw, legacy_site, x, y, db_ref) %>%
+  arrange(grts_ranking_draw, point_code)
+```
+
+Als voorbeeld tonen we de eerste zes rijen van de tabel.
+
+```{r}
+mhq_streams_refpoints %>%
+  head() %>%
+  kable() %>%
+  kable_styling()
+```
+
+
+```{r}
+write_vc(mhq_streams_refpoints, file = "mhq_streams_refpoints", root = "../output", sorting = c("sampling_unit_code", "point_code"), strict = FALSE)
+```
+
+### mhq_streams_refpoints_validity
+
+De tabel 'mhq_streams_refpoints_validity' geeft aan welke steekproefeenheden voldoen aan de voorwaarden om opgenomen te worden in het biotisch meetnet habitatkwaliteit voor 3260.
+
+De tabel bestaat uit volgende variabelen:
+
++ scheme
++ sampling_unit_code
++ point_code
```

Comment:
the point code is defined in the refpoints dataframe; should not be repeated here. Each dataframe must have unique key (here: scheme + sampling unit) + unique attributes (not repeated elsewhere), to keep maintenance minimal and feed a database in a consistent way later. See general comment at the top.